File: my-folder/1894-merge-strings-alternately/solution.cpp
class Solution {
public:
    string mergeAlternately(string a, string b) {
        string ans = "";
        int n = max(a.length(), b.length());
        for(int i=0; i<n; i++) {
            if(i<a.length()) ans += a[i];
            if(i<b.length()) ans += b[i];

        }

        return ans;
        
    }
};

File: my-folder/1894-merge-strings-alternately/solution.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        n = len(word1)
        m = len(word2)

        minn = min(n, m)
        ans = ""
        for i in range(minn):
          ans += word1[i]
          ans += word2[i]
        if n > m:
          ans += word1[minn::]
        else:
          ans += word2[minn::]
        return ans            
